documentation/getting-started.md

```diff
@@ -10,47 +10,31 @@ redirect_from:
+To access all these features, you will need to install the [WoTT Agent and Dashboard](https://dash.wott.io) to manage your Edge/IoT devices through a simple and user-friendly interface. 
 
-Ultimately we want to give you the developers the safety of secure devices on the internet so you can integrate your projects on a multitude of platforms without the complexity of managing individual certificates or authentication. WoTT secures your projects so that you don't have to.
-
-## A little more on 'cryptographic' identity
-
-Our lightweight agent handles identifying your device as well as verifying inter-device communication cryptographically [1]. What this essentially means is that the agent assigns a certificate from WoTT's own certificate authority (CA) to your device to secure it [2].
-The agent does this based on a unique value assigned to your device which acts as an identifier to other devices. The purpose of this is to restrict access to trusted devices preventing fraudulent access and attacks.
-
-* [1] Done through mutual TLS (mTLS)
-* [2] WoTT does this using an x509 security certificate. These certificates refresh every 7 days.
-
-The certificates are managed entirely by WoTT - the only thing you need to do to begin securing your device is install the WoTT Agent.
-
-## Credentials and the WoTT Dashboard
-
-In an ideal world, we would do away with the need for usernames and passwords entirely and rely on cryptographic identity alone. However, servers cannot verify identity from browsers via cryptographic identity (yet). Servers are unable to verify who you are which is why you are required to login to access for example your email or Google cloud services. The same applies to Web Applications that you as an IoT developer create which can be accessed from any browser.
-
-As a developer, you will likely be accessing a large range of internet/cloud services that require basic HTTP authentication or an API key. In most instances, such credentials are hard coded into applications and can be easily read with access to said applications. WoTT introduces credential management so that you can remotely download the necessary credentials onto your device as a JSON file. This removes the need for hard coded keys and logins from your application.
-
-We do this via our Dashboard which you can use to monitor your devices. The Dashboard gives your device a trust score (based on how secure it is) as well as giving you the means to add and manage credentials for whatever purpose you need them for.
+We'll provide a step-by-step walkthrough of set-up and installation of the WoTT agent. It's hassle-free and simple to implement.
 
 ## Installing the WoTT agent and claiming your device
```

Comment:
`s/device/node`

Reply:
I'm confused by this

Reply:
@fshmcallister you not fluent in regular expression? ;)

Replace device with node in the article

Reply:
bully :P

Reply:
how come node though?

Reply:
We're tweaking the messages to be less IoT oriented and more towards servers. Node is a nice catch-all. We're toying with words like 'node', 'endpoint', 'system'. Happy to get you input if you can think of any other nice catch-all.

Reply:
Nope no problem! Let me reword a little more then since getting started is still quite IoT focused